eperusteet/eperusteet-service/src/main/java/fi/vm/sade/eperusteet/service/OsaamismerkkiService.java

```diff
@@ -0,0 +1,41 @@
+package fi.vm.sade.eperusteet.service;
+
+import fi.vm.sade.eperusteet.dto.osaamismerkki.OsaamismerkkiBaseDto;
+import fi.vm.sade.eperusteet.dto.osaamismerkki.OsaamismerkkiDto;
+import fi.vm.sade.eperusteet.dto.osaamismerkki.OsaamismerkkiKategoriaDto;
+import fi.vm.sade.eperusteet.dto.osaamismerkki.OsaamismerkkiQuery;
+import org.apache.tika.mime.MimeTypeException;
+import org.springframework.data.domain.Page;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.HttpMediaTypeNotSupportedException;
+
+import java.util.List;
+
+public interface OsaamismerkkiService {
+    @PreAuthorize("permitAll()")
+    List<OsaamismerkkiBaseDto> findJulkisetBy(OsaamismerkkiQuery query);
+
+    @PreAuthorize("permitAll()")
+    Page<OsaamismerkkiDto> findBy(OsaamismerkkiQuery query);
+
+    @PreAuthorize("permitAll()")
+    OsaamismerkkiBaseDto getJulkinenOsaamismerkki(Long id);
+
+    @PreAuthorize("permitAll()")
+    List<OsaamismerkkiKategoriaDto> getJulkisetKategoriat();
+
+    @PreAuthorize("hasPermission(null, 'perusteprojekti', 'LUONTI')")
```

Comment:
Pitikös näitä pystyä hallinnoimaan vain oph admin tunnuksilla?
Tällöin 'perusteprojekit' pitää vaihtaa 'pohja'